<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calc</class>
 <widget class="QMainWindow" name="Calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="3">
     <widget class="QPushButton" name="Button2">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #C0C0C0; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="Button9">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #C0C0C0; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Button14">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #C0C0C0; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>E</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="Dec">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #87CEEB; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed
 {
    background-color: #1E90FF; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>Dec</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="Button6">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #C0C0C0; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="Button0">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #C0C0C0; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="Oct">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #87CEEB; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed
 {
    background-color: #1E90FF; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>Oct</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="Hex">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #87CEEB; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed
 {
    background-color: #1E90FF; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>Hex</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="Button8">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #C0C0C0; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Button13">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #C0C0C0; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Button4">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #C0C0C0; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Button11">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #C0C0C0; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Button10">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #C0C0C0; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="Button3">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #C0C0C0; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="Bin">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #87CEEB; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed
 {
    background-color: #1E90FF; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>Bin</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="Button12">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #C0C0C0; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Button7">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #C0C0C0; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="Float">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #87CEEB; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed
 {
    background-color: #1E90FF; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>Float</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="Button5">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #C0C0C0; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Button15">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #C0C0C0; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>F</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Button1">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #C0C0C0; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="Mode">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit 
{
     background-color: gray; 
	border: 1px solid gray;
	color: #ffffff;
}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>48</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit 
{
     background-color: gray; 
	border: 1px solid gray;
	color: #ffffff;
}</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="Divide">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #FA8072; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #E9967A; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="Multiply">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #FA8072; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #E9967A; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="Add">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #FA8072; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #E9967A; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="Substract">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #FA8072; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #E9967A; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QPushButton" name="Equals">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #FA8072; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #E9967A; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="Clear">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #FA8072; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #E9967A; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="ChangeSign">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
     background-color: #FA8072; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed 
{
    background-color: #E9967A; 
	border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
